hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/rmadmin/FederationRMAdminInterceptor.java

```diff
@@ -71,10 +71,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.util.List;
-import java.util.Map;
-import java.util.Collection;
-import java.util.Set;
+import java.util.*;
```

Comment:
Avoid

Reply:
Thanks for helping to review the code! I will fix it.